code/game/objects/structures/maintenance.dm

```diff
@@ -74,9 +78,8 @@ at the cost of risking a vicious bite.**/
 	if(critter_infested && prob(50) && iscarbon(user))
 		var/mob/living/carbon/bite_victim = user
 		var/obj/item/bodypart/affecting = bite_victim.get_bodypart("[(user.active_hand_index % 2 == 0) ? "r" : "l" ]_arm")
+		to_chat(user, span_danger("You feel a sharp as an unseen creature sinks it's [pick("fangs", "beak", "proboscis")] into your arm!"))
```

Comment:
You feel a sharp?

Reply:
Wow... the old message said that? How did I not notice, hahaha